grpc/server/src/engine_server/mappings/ipc/step.rs

```diff
@@ -189,6 +189,16 @@ impl TryFrom<StepRequest> for ipc::StepRequest {
         };
         result.set_reward_items(reward_items.into());
 
+        let evict_items = {
+            let mut ret: Vec<ipc::EvictItem> = vec![];
+            for item in step_request.evict_items.into_iter() {
+                let ipc = item.try_into()?;
+                ret.push(ipc);
+            }
+            ret
+        };
```

Comment:
Maybe more concise:
```rust
let evict_items = step_request
    .evict_items
    .into_iter()
    .map(TryInto::try_into)
    .collect::<Result<Vec<_, _>>>()?;
```
What is the vector turned `into()` in the next line? Can we even avoid `Vec` allocation?
(Anyway, none of this is important, of course!)

Reply:
Agreed, but this is not my code...just coming from the merge.

Reply:
Right, sorry!